src/components/dataset/datasetInformation/DatasetInformation.tsx

```diff
@@ -0,0 +1,164 @@
+import Image from 'next/image'
+import { Button, Icon, Typography, useTheme, useTranslation } from '@okp4/ui'
+import type { DeepReadonly, ThemeContextType, UseTranslationResponse } from '@okp4/ui'
+import './datasetInformation.scss'
+import { formatDate } from '../../../utils'
+import type {
+  Dataset,
+  DatasetGovernance
+} from '../../../pages/dataspace/[dataspace-id]/dataset/[dataset-id]'
+import type { TFunction } from 'react-i18next'
+
+type DatasetInformationProps = {
+  readonly dataset: Dataset
+}
+
+type BackgroundImageProps = {
+  readonly url: string
+}
+
+type ContainerProps = {
+  readonly children: React.ReactNode
+  readonly name: string
+}
+
+type GovernanceProps = {
+  readonly governance: DatasetGovernance
+  readonly theme: string
+  readonly t: TFunction
+}
+
+type MetadataProps = {
+  readonly dataset: Dataset
+  readonly theme: string
+}
+
+type Metadata = {
+  readonly size: number
+  readonly format: string
+  readonly quality: number
+  readonly completeness: number
+  readonly provider: string
+  readonly updatedOn: string
+}
+
+type MetadataRowProps = {
+  readonly children: React.ReactNode
+  readonly name: string
+  readonly unit?: string
+}
+
+const BackgroundImage = ({ url }: DeepReadonly<BackgroundImageProps>): JSX.Element => (
+  <div className="okp4-dataset-background-image">
+    <Image alt="dataset background image" layout="fill" objectFit="cover" src={url} />
+  </div>
+)
+
+const Container = ({ children, name }: DeepReadonly<ContainerProps>): JSX.Element => (
+  <div className="okp4-dataset-information-container">
+    <Typography as="h2" color="inverted-text" fontSize="small" fontWeight="bold">
+      {name}:
+    </Typography>
+    {children}
+  </div>
+)
+
+const Governance = ({ governance, theme, t }: DeepReadonly<GovernanceProps>): JSX.Element => (
+  <div className="okp4-dataset-governance">
+    <Typography as="p" color="inverted-text" fontSize="small">
+      {t('dataset:governance:based', { name: governance.name, token: governance.based })}
+    </Typography>
+    <Button
+      backgroundColor={theme === 'dark' ? 'secondary' : 'primary'}
```

Comment:
Is this a "hack" to prevent from the orange color?

Reply:
Yes

Reply:
OK
But could you please add a `TODO` comment that explains this should be removed with the next evolutions to come in the `okp4/ui` library?
Thxs